### PR TITLE
Error catching & ruby version definition

### PR DESCRIPTION
**Pull request by iloveitaly**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,4 +1,5 @@
 source 'https://rubygems.org'
+ruby '2.1.5'
```
Comment (phstc) [inline Gemfile]: hey @iloveitaly 

Our servers are running ruby 2.1.2 we still need to update them. Could you drop this line?

Comment (iloveitaly) [inline Gemfile]: > Our servers are running ruby 2.1.2 we still need to update them. Could you drop this line?

Can I downgrade to 2.1.2 instead?

My CI tests won't run without an explicit ruby definition

Comment (phstc) [inline Gemfile]: hm

We are planning to upgrade our containers ruby soon. So I would prefer to drop this line. Any chance you can set this value in a `.ruby_version` file or CI specific settings?

Comment (iloveitaly) [inline Gemfile]: Done. Check out the updated branch.

**Comment by phstc** [comment]
Added to master & deployed :beer:

**Pull request merged**